themes/default-bootstrap/js/cart-summary.js

```diff
@@ -46,7 +46,7 @@ $(document).ready(function(){
 	});
 
 	$('.cart_quantity_input').typeWatch({
-		highlight: true, wait: 600, captureLength: 0, callback: function(val){
+		highlight: true, wait: 600, captureLength: 1, callback: function(val){
 			updateQty(val, true, this.el);
```

Comment:
Hi,

With the new version of typewatch, ```this.el``` must be replaced by ```this```.

Best regards